FirebaseFunctions.podspec

```diff
@@ -0,0 +1,33 @@
+#
+# Be sure to run `pod lib lint FirebaseFunctions.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseFunctions'
+  s.version          = '1.0.0'
+  s.summary          = 'Cloud Functions for Firebase iOS SDK.'
+
+  s.description      = <<-DESC
+iOS SDK for Cloud Functions for Firebase.
+                       DESC
+
+  s.homepage         = 'https://developers.google.com/'
+  s.authors          = 'Google, Inc.'
+  s.source           = { :git => 'https://github.com/TBD/FirebaseFunctions.git', :tag => s.version.to_s }
```

Comment:
We should update this URL. Maybe not in this PR.

Reply:
Let's do it in a different PR. It's not needed until we switch to source pods.